tests/spdx3/model/fixtures.py

```diff
@@ -0,0 +1,428 @@
+from spdx_tools.spdx3.model.external_identifier import ExternalIdentifier, ExternalIdentifierType
+from spdx_tools.spdx3.model.external_map import ExternalMap
+from spdx_tools.spdx3.model.external_reference import ExternalReference, ExternalReferenceType
+from spdx_tools.spdx3.model.hash import Hash, HashAlgorithm
+from spdx_tools.spdx3.model.namespace_map import NamespaceMap
+from spdx_tools.spdx3.model.organization import Organization
+from spdx_tools.spdx3.model.person import Person
+from spdx_tools.spdx3.model.relationship import Relationship, RelationshipCompleteness, RelationshipType
+from spdx_tools.spdx3.model.software_agent import SoftwareAgent
+from spdx_tools.spdx3.model.spdx_document import SpdxDocument
+from spdx_tools.spdx3.model.tool import Tool
+
+"""Utility methods to create data model instances. All properties have valid defaults, so they don't need to be
+specified unless relevant for the test."""
+
+
+def creation_info_fixture(
+    spec_version=Version("3.0.0"),
+    created=datetime(2022, 12, 1),
+    created_by=["creatorCreationInfo"],
+    created_using=["createdCreationInfo"],
+    profile=["profileCreationInfo"],
+    data_license="CC0-1.0",
+    comment="commentCreationInfo",
+) -> CreationInformation:
+    return CreationInformation(
+        spec_version=spec_version,
+        created=created,
+        created_by=created_by,
+        created_using=created_using,
+        profile=profile,
+        data_license=data_license,
+        comment=comment,
+    )
+
+
+def external_identifier_fixture(
+    external_identifier_type=ExternalIdentifierType.OTHER, identifier="identifier_ext_iden", comment="comment_ext_iden"
+) -> ExternalIdentifier:
+    return ExternalIdentifier(external_identifier_type, identifier, comment)
+
+
+def external_reference_fixture(
+    external_reference_type=ExternalReferenceType.OTHER,
+    locator=None,
+    content_type="content_type_exter_ref",
+    comment="comment_exter_ref",
+) -> ExternalReference:
+    locator = [] if locator is None else locator
+    return ExternalReference(external_reference_type, locator, content_type, comment)
+
+
+def hash_fixture(algorithm=HashAlgorithm.SHA1, hash_value="hash_value", comment="comment_hash_algorithm") -> Hash:
+    return Hash(algorithm=algorithm, hash_value=hash_value, comment=comment)
+
+
+def external_map_fixture(
+    external_id="https://spdx/tools-python/ExternalMapFixture",
+    verified_using=[hash_fixture()],
+    location_hint="https://spdx/tools-python/location_hint_ExternalMap",
+) -> ExternalMap:
+    verified_using = [] if verified_using is None else verified_using
+    return ExternalMap(external_id=external_id, verified_using=verified_using, location_hint=location_hint)
+
+
+def namespace_map_fixture(prefix="prefix_namespace_map", namespace="namespace_namespace_map") -> NamespaceMap:
+    return NamespaceMap(prefix=prefix, namespace=namespace)
+
+
+def agent_fixture(
+    spdx_id="https://spdx/tools-python/AgentFixture",
```

Comment:
```suggestion
    spdx_id="https://spdx.test/tools-python/AgentFixture",
```
This indicates that the URI is only for test purposes, would also apply to all other spdx_ids.